src/components/network-chart/index.css

```diff
@@ -90,14 +80,11 @@
 .networkChartRowItemUriRequired {
   display: inline-block;
   overflow: hidden;
+  max-width: 600px;
+  text-overflow: ellipsis;
+  white-space: nowrap;
```

Comment:
Some "required" parts can be very long: try to search for "quant" on the master link, and compare with the deploy preview, to know what I mean here.

Reply:
This is much nicer, thanks. Should the max width be a percentage here? The 600px feels a bit arbitrary, I could be looking at this profile on a large screen.

Also, a potential follow-up I'm seeing is that the URL isn't fully visible in the tooltip. I'm not sure what's going on.

<img width="656" alt="image" src="https://user-images.githubusercontent.com/1588648/56663966-dbb3b700-666c-11e9-9572-f1ae09989aff.png">

Reply:
I believe we want `overflow-wrap: break-word;`, I can add it in this PR if that's enough to make it work.

Reply:
ooops I forgot to use a %age before landing, I'll do a follow-up before forgetting.